aws/ecs/inputs.tf

```diff
@@ -135,4 +135,8 @@ variable "unmasked_appstore_tag" {
 
 variable "aggregate_metrics_table_arn" {
   type = string
-}
\ No newline at end of file
+}
+
+variable "default_schedule_expression" {
```

Comment:
Can we drop the word default from this variable, since it's not the default it's just the schedule_expression